src/components/reusable/table/table-toolbar.scss

```diff
@@ -0,0 +1,19 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  padding: var(--kd-spacing-8) var(--kd-spacing-0);
```

Comment:
Don't think the toolbar needs top padding, since it is the topmost part of the table.

Reply:
The toolbar design is just a rough version, and there's top and bottom padding in it.
We can update this later when the design for it gets revamped and completed.